docs/source/commands.rst

```diff
@@ -33,3 +35,39 @@ Command-line options
 .. option:: --app-label <app_label>
 
    Create initial data with the *app_label* provided.
+
+.. _command-generate-pep-pages:
+
+generate_pep_pages
+------------------
+
+This Django management command generates `pages.Page` objects from the output
+of the existing PEP repository generation process. You run it like::
+
+    $ ./manage.py generate_pep_pages
+
+To get verbose output, specify :option:`--verbosity` option::
+
+    $ ./manage.py generate_pep_pages --verbosity=2
+
+It uses the conversion code in the `peps.converters` module in an attempt to
+normalize the formatting for display purposes.
+
+Command-line options
+^^^^^^^^^^^^^^^^^^^^
+
+.. program:: manage.py generate_pep_pages
+
+.. option:: --verbosity=<verbosity_value>
+
+   Get verbose output when *verbosity_value* > 1.
+
+.. _command-dump-pep-pages:
+
+dump_pep_pages
+--------------
+
+This simply dumps our PEP related pages as JSON. The `dumpdata` content is
```

Comment:
This simply -> This command simply

Reply:
It would be better to not mention about the dumpdata command here. `dump_pep_pages` is much simpler than dumpdata. Perhaps just say that it dumps everything to `sys.stdout`?